extensions/rocksdb-repos/FlowFileRepository.cpp

```diff
@@ -156,14 +156,12 @@ void FlowFileRepository::prune_stored_flowfiles() {
         search->second->put(eventRead);
       } else {
         logger_->log_warn("Could not find connection for %s, path %s ", eventRead->getConnectionUuid(), eventRead->getContentFullPath());
-        if (eventRead->getContentFullPath().length() > 0) {
-          if (nullptr != eventRead->getResourceClaim()) {
-            content_repo_->remove(eventRead->getResourceClaim());
-          }
-        }
+        auto claim = eventRead->getResourceClaim();
+        if (claim) claim->decreaseFlowFileRecordOwnedCount();
```

Comment:
Isn't this going to leave content repo entries behind?

Reply:
here we decrement on behalf of the persisted FlowFile we just read from the repo, then mark the FlowFile for deletion